spec/definitions/AppVersionDelete.yaml

```diff
@@ -0,0 +1,20 @@
+type: object
```

Comment:
Why is app version delete is linked to a template ? And if it's not the same model for create/update or delete. It's better to mention whether this is used for request for response.

Reply:
This is a specific edge case @tarun91vas 
@abdulMuneerM  will be able to answer why its the case.